### PR TITLE
feat(build): replace build with webpack

### PR DESCRIPTION
**Pull request by ej9x**

### CODE REVIEW

File: packages/api-client/package.json [hunk 1]
```diff
@@ -28,11 +28,12 @@
   },
   "dependencies": {
     "@8base/error-codes": "^1.0.5",
+    "@8base/webpack-configuration": "0.3.5",
```
Comment (vorobeez) [inline packages/api-client/package.json]: i think webpack-configuration should be a dev dependency.

Comment (ej9x) [reply packages/api-client/package.json]: fixed

Comment (ej9x) [reply packages/api-client/package.json]: fixed

**Comment by vorobeez** [comment]
@ej9x Why did you delete all yarn.locks?

**Pull request merged**